service/org.nemomobile.BtrfsBalancer.service

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
+Interface=org.nemomobile.BtrfsBalancer
+Name=org.nemomobile.BtrfsBalancer
+Exec=/usr/sbin/btrfs-balancer server
+User=root
```

Comment:
We should have also systemd service for this, that has the requirements. I'm assuming that this requires: After=local-fs.target at least. See https://github.com/nemomobile/ssu/pull/35/files for a reference how to map dbus with systemd services.